lib/DDG/Spice/LeakDB.pm

```diff
@@ -22,7 +22,7 @@ triggers startend => 'hashme', 'leakdb';
 spice to => 'http://api.leakdb.abusix.com/?j=$1';
 spice wrap_jsonp_callback => 1;
 
-handle remainder_lc => sub{
+handle remainder => sub{
```

Comment:
Nevermind, I see now that this IA has actually been broken for a long time because of the `_lc`! We've been sending the API the **lowercased** input all this time...

We can actually remove that regex as I believe it's not needed at all. We're dealing with the remainder of the query, which is expected to be just a single "word". We should really check that there are no spaces in the remainder and if not `return $_`.